Added Ubuntu PPA description (#6)
goxray/desktop: README.md

#### Tested and supported on:
- macOS (tested on Sequoia 15.1.1 M3)
- Linux:
    - Ubuntu 24.10 ARM

>     - Mint Linux 22 amd64

Reviewer: ```suggestion
    - Mint 22 amd64
```

> sudo setcap cap_net_raw,cap_net_admin,cap_net_bind_service+eip goxray_binary_path
> ```

> ##### Ubuntu PPA
> 
> The Ubuntu PPA version is based on the tagged release of this project and named `goxray-gui`. It is automatically built by Ubuntu build pipeline from the packaged sources. The preparation and version control pipelines for source preparation live in the [dedicated repository](https://github.com/twdragon/xray-debian-pkg). To install the client program from the PPA, issue the following commands:

Reviewer: Let's rewrite it a bit to something like this:

```suggestion
##### 📦 Using [twdragon](https://github.com/twdragon) debian package

The client is available from the PPA repository `ppa:twdragon/xray`, maintained by [twdragon](https://github.com/twdragon). The package is in sync with this repo releases. You can check the pipeline at the [dedicated repository](https://github.com/twdragon/xray-debian-pkg). To install use:
```

```bash
sudo add-apt-repository ppa:twdragon/xray
sudo apt update
sudo apt install goxray-gui
```

> After the installation, the package might be updated automatically as it is done in Ubuntu. All the packages in this PPA are cryptographically signed by [twdragon](https://github.com/twdragon). The PPA is also available on Ubuntu Launchpad: https://launchpad.net/~twdragon/+archive/ubuntu/xray
> 
> The experimental unsigned artifact packages of the program could be also found in the Github Action Artifacts section of the [pipeline repository](https://github.com/twdragon/xray-debian-pkg/actions).

Reviewer: ```suggestion
After the installation, the package might be updated automatically as it is done in Ubuntu. Packages are signed by [twdragon](https://github.com/twdragon) and published on [Launchpad](https://launchpad.net/~twdragon/+archive/ubuntu/xray). Experimental builds are also available in [pipeline repository](https://github.com/twdragon/xray-debian-pkg/actions).
```
